tests/apollo/util/skvbc_history_tracker.py

```diff
@@ -844,7 +848,10 @@ async def get_last_block_id(self, client):
         return kvbc.SimpleKVBCProtocol.parse_reply(await client.read(msg))
 
     async def send_tracked_write(self, client, max_set_size, long_exec=False):
-        readset = self.readset(0, max_set_size)
+        max_read_set_size = max_set_size
+        if self.no_conflicts:
+            max_read_set_size = 0
```

Comment:
A more "pythonistic" way to do this would be to say:
```python
max_read_set_size = 0 if self.no_conflicts else max_set_size
```

Reply:
done